app/webtrc.go

```diff
@@ -1,28 +0,0 @@
-// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This had me completely baffled for longer than I want to admit. I just couldn't figure out why `ag -Q 'HttpClient()'` was saying there was an instance of 'HttpClient()' on line 27 of `app/webrtc.go`. Eventually I realized that wasn't quite what it was saying...